packages/qwik/src/core/shared/error/error.ts

```diff
@@ -51,7 +51,7 @@ export const codeToText = (code: number, ...parts: any[]): string => {
       'Unknown vnode type {{0}}.', // 43
       'Materialize error: missing element: {{0}} {{1}} {{2}}', // 44
       'Cannot coerce a Signal, use `.value` instead', // 45
-      '', // 46 unused
+      'useComputedSignal$ QRL {{0}} {{1}} returned a Promise', // 46
```

Comment:
maybe this will make it more clear to the developer?

"useComputedSignal$ QRL {{0}} {{1}} cannot return a Promise"